tokio-trace/tokio-trace-core/src/callsite.rs

```diff
@@ -67,24 +88,27 @@ pub struct Identifier(
     pub &'static Callsite,
 );
 
+/// Clear and reregister interest on every [`Callsite`]
+///
+/// This function is intended for runtime reconfiguration of filters on traces
+/// when the filter recalculation is much less frequent than trace events are.
+/// The alternative is to have the Subscriber that supports runtime reconfiguration
+/// always return Interest::sometimes() and manually reimplement the filtering
+/// of the events it receives.
+///
+/// [`Callsite`]: ../callsite/trait.Callsite.html
+pub fn invalidate_interest() {
```

Comment:
I may name this something like “recompute_cache”, “rebuild_callsite_cache”, ... some name that invoked that logic is executed to regenerate cache data.